modules/dcache-nfs/src/main/java/org/dcache/chimera/nfsv41/door/NFSv41Door.java

```diff
@@ -858,6 +858,34 @@ public String call() throws IOException {
         }
     }
 
+
+    @Command(name = "show transfers filtered", hint = "show active transfers filtered",
+            description = "Show active transfers excluding proxy-io filtered by pool, client or pnfsid.")
+    public class ShowTransfersFilteredCmd implements Callable<String> {
+
+        @Argument(required = false, metaVar = "pool")
```

Comment:
Although arguments can be made optional, there's an inherent problem -- arguments are ordered, which means that an admin cannot filter on pnfsid without also filtering on client and pool.

One way around this (what we do elsewhere) is to use `@Option` annotation instead.  This gives the admin the opportunity to "switch on" the desired behaviour; for example:

    show transfers -pool=my_interesting_pool -pnfsid=....